### DISCUSSION TITLE
Request: Please Add a "Filtering overrides" options section

### DISCUSSION THREAD

**Discussion by awesomeocelot** [uBlockOrigin/uBOL-home#175] [Ideas]
This could be implemented similar to the "No filtering" section in the existing uBOL extension or as separate options tabs like the "Trusted sites" options tab in your original Ublock Origin extension. The issue is that there is no way to read or edit the list of overidden websites for Basic vs Optimal vs Complete filtering from whatever the user has set the default filtering level to in the uBOL Options GUI, only the "No filtering" overrides can be read and edited but not different non-default filtering level overrides.

This means that the only option to reset all websites to the "default" filtering level is to delete and re-install the entire extension, which seems excessive.

**Comment by garry-ut99** [comment]
Related to: https://github.com/uBlockOrigin/uBOL-home/issues/158#issuecomment-2266843804 + https://github.com/uBlockOrigin/uBOL-home/issues/158#issuecomment-2266900154

**Comment by gorhill** [comment]
https://github.com/uBlockOrigin/uBOL-home/issues/270
